### PR TITLE
feat(shared-docs): move embedded editor to @angular/docs

### PR DESCRIPTION
**Pull request by josephperrott**
Move embedded editor code to shared code location.

### CODE REVIEW

File: docs/editor/code-editor/code-editor.component.scss [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+  --code-tags: var(--bright-blue);
+  --code-component: var(--primary-contrast);
+  --code-type-name: var(--vivid-pink);
+  --code-self: var(--orange-red);
+
+  position: relative;
+}
+
+.adev-code-editor-tabs {
```
Comment (jelbourn) [inline docs/editor/code-editor/code-editor.component.scss:68]: I don't think we should do this now, but at some point we're going to want to replace these "adev" prefixes with "docs"

**Review by jelbourn — APPROVED** [review]
LGTM, assuming all (most?) of the TS code is copied from blossom

File: docs/editor/code-editor/code-editor.component.scss (hunk 1 above)
Comment (josephperrott) [reply docs/editor/code-editor/code-editor.component.scss:68]: Yeah, agreed, there is a ton of that type of cleanup that needs to happen.

**Comment by josephperrott** [comment]
This PR was merged into the repository by commit 3bdb358b99b17910167b2d663088b4f1d5d0ca09.

**Pull request closed without merging**